ctapipe/image/modifications.py

```diff
@@ -0,0 +1,113 @@
+from abc import abstractmethod
+import numpy as np
+from ..core.component import TelescopeComponent
+from ..core.traits import FloatTelescopeParameter, BoolTelescopeParameter
+
+
+def add_noise(image, noise_level, rng=None, correct_bias=True):
+    """
+    Create a new image with added poissonian noise
+    """
+    if not rng:
+        rng = np.random.default_rng()
+    noisy_image = image.copy()
+    noise = rng.poisson(noise_level)
+    noisy_image += noise
+    if correct_bias:
+        noisy_image -= noise_level
+    return noisy_image
+
+
+def smear_image(image, geom, smear_factor):
+    """
+    Create a new image with values smeared to the direct pixel neighbors
+    Pixels at the camera edge lose charge this way.
+    """
+    # make clear what smear factor is supposed to mean and that only direct neighbors are taken into account
+    # smear factor can be an array as well -> selection in tool possible!
+    # a more sophisticated approach might make use of the pixel area, but thats complicated
+    if geom.pix_type.value == "hexagon":
+        max_neighbors = 6
+    elif geom.pix_type.value == "rectangular":  # thats probably labeld differently
+        max_neighbors = 8  # or 4? lookup neighbor matrix for a rect cam
```

Comment:
I think we should use a gaussian kernel here. E.g. not use the same amount for the direct neighbors as for the diagonal neighbors. I don't know how easy that is though.

Reply:
I will have a look. It seemed to be a bit tricky, but I havent looked very closely

Reply:
So direct neighbors only, but there is a constant factor between diagonal and non-diagonal pixels?
That should be easy. What would be tricky, would be to incorporate the pixel size and also smear to pixels beyond the first neighbor.